mobile-app/app/screens/AppNavigator/screens/Balances/components/TokenBreakdownPercentage.tsx

```diff
@@ -0,0 +1,140 @@
+import { View } from '@components'
+import { SymbolIcon } from '@components/SymbolIcon'
+import { TextSkeletonLoader } from '@components/TextSkeletonLoader'
+import { ThemedIcon, ThemedText, ThemedView } from '@components/themed'
+import { RootState } from '@store'
+import { tailwind } from '@tailwind'
+import { translate } from '@translations'
+import BigNumber from 'bignumber.js'
+import { TouchableOpacity } from 'react-native'
+import NumberFormat from 'react-number-format'
+import { useSelector } from 'react-redux'
+import { useTokenBreakdownPercentage } from '../hooks/TokenLockedBalance'
+
+interface TokenBreakdownPercentageProps {
+  lockedAmount: BigNumber
+  availableAmount: BigNumber
+  symbol: string
+  onBreakdownPress: () => void
+  isBreakdownExpanded: boolean
+}
+
+export function TokenBreakdownPercentage (props: TokenBreakdownPercentageProps): JSX.Element {
+  const {
+    availablePercentage,
+    lockedPercentage
+  } = useTokenBreakdownPercentage({ available: props.availableAmount, locked: props.lockedAmount })
+  const { hasFetchedToken } = useSelector((state: RootState) => state.wallet)
+
+  return (
+    <View style={tailwind('flex flex-row items-center bg-transparent justify-between mb-3')}>
+      <View style={tailwind('flex flex-row items-center flex-wrap flex-1')}>
+        {hasFetchedToken
```

Comment:
can we add `hasFetchedVaultsData` also, as locked value is depend upon vault data

Reply:
added